tests/pages/activity.js

```diff
@@ -16,5 +16,7 @@ export default create({
   typeLetter: typeLetter('.activity-script-viewer'),
   completed: text('.activity-script-viewer .completed'),
   pending: text('.activity-script-viewer .pending'),
-  activityCompletedMessage: text('.activity-completed')
+  activityCompletedMessage: text('.activity-completed'),
+  restart: clickable('.btn-restart'),
+  restartHasFocus: $(document.activeElement).hasClass('btn-restart')
```

Comment:
I think this implementation is not right. The problem is that we need page object properties to be evaluated lazily.

In order to achieve this we need to define a function instead.

```js
restarHasFocus: function() { return $(document.activeElement).hasClass('btn-restart') }
```

Another way of achieving the same but without using a function is to declare a descriptor.

```
restartHasFocus: {
  isDescriptor: true,
  get: function() {
    return $(document.activeElement).hasClass('btn-restart');
  }
}
```

Here's an example of how to declare a descriptor https://github.com/san650/ember-cli-page-object/blob/b9a36f01a8b3d265c7a14aa6bac29e4260d08e8c/test-support/page-object/predicates/has-class.js#L83-L95